sdc/datatypes/hpat_pandas_dataframe_functions.py

```diff
@@ -983,11 +979,8 @@ def sdc_pandas_dataframe_isna_codegen(df, func_name, params, ser_params):
 
 def df_index_codegen_all(self):
     if isinstance(self.index, types.NoneType):
-        func_lines = ''
-    else:
-        func_lines = ', index = df._index'
-
-    return func_lines
+        return ''
+    return 'index=df._index'
```

Comment:
This hardcodes the name of the df param, not sure if it's worth fixing this.

Reply:
This is a temporary solution until we fix the creation of a dataframe with the None index.